apps/people-app/backend/modules/db_script/people.sql

```diff
@@ -0,0 +1,346 @@
+SET FOREIGN_KEY_CHECKS = 0;
+
+DROP DATABASE IF EXISTS `hris_people`;
+CREATE DATABASE `hris_people`;
+USE `hris_people`;
+
+SET FOREIGN_KEY_CHECKS = 1;
+
+-- Org structure tables
+CREATE TABLE `business_unit` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(45) NOT NULL,
+  `head_email` VARCHAR(254) NOT NULL,
+  `created_by` VARCHAR(254) NOT NULL,
+  `created_on` TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6),
+  `updated_by` VARCHAR(254) NOT NULL,
+  `updated_on` TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6),
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
```

Comment:
Please do this to all.
```suggestion
) 
```